dataprofiler/data_readers/data_utils.py

```diff
@@ -31,7 +43,9 @@ def data_generator(data_list):
         yield item
 
 
-def generator_on_file(file_object):
+def generator_on_file(
+    file_object: Union[StringIO, BytesIO]
+) -> Generator[Union[str, bytes], None, None]:
```

Comment:
is this after `black` formatting pre-commit?

Reply:
yes